### DISCUSSION TITLE
Automatic transcription

### DISCUSSION THREAD

**Discussion by Subtitle-Bob** [Purfview/whisper-standalone-win#176] [Q&A] [answered]
Is there a way to run the program automatically so that new files (e.g. mp4) in a folder are automatically transcribed one after the other?

**Comment by Purfview** [comment, marked as answer]
Use Windows Task Scheduler:

https://www.backup4all.com/how-to-create-a-new-task-using-windows-task-scheduler-kb.html

https://www.youtube.com/watch?v=ON2uRHlt_GI

**Reply by Subtitle-Bob** [reply]
It can be so easy... Thanks!
